lasy/profiles/transverse/gaussian_profile.py

```diff
@@ -19,11 +19,28 @@ class GaussianTransverseProfile(TransverseProfile):
     ----------
     w0 : float (in meter)
         The waist of the laser pulse, i.e. :math:`w_0` in the above formula.
+
+    wavelength : float (in meter)
+        The main laser wavelength :math:`\\lambda_0` of the laser.
+
+    z_init : float (in meter), optional
```

Comment:
This name sounds good to me, as would the now traditional `z_foc`. Any other opinion?

Reply:
Discussed offline: yes, we will change this to `z_foc`, and mention that the equivalent `propagate` call would be `.propagate(-z_foc)`